ui/webui/src/components/review/ReviewConfiguration.jsx

```diff
@@ -162,58 +121,61 @@ export const ReviewConfiguration = ({ deviceData, diskSelection, language, reque
 
     return (
         <AnacondaPage title={_("Review and install")}>
-            <ExpandableSection
-              className="review-expandable-section"
-              id={`${idPrefix}-language`}
-              toggleText={<Title headingLevel="h3">{_("Language")}</Title>}
-              onToggle={() => setShowLanguageSection(!showLanguageSection)}
-              isExpanded={showLanguageSection}
-              isIndented
-            >
-                <ReviewDescriptionList>
-                    <DescriptionListGroup>
-                        <DescriptionListTerm className="description-list-term">
-                            {_("Language")}
-                        </DescriptionListTerm>
-                        <DescriptionListDescription className="description-list-description" id={idPrefix + "-target-system-language"}>
-                            {language["native-name"].v}
-                        </DescriptionListDescription>
-                    </DescriptionListGroup>
-                </ReviewDescriptionList>
-            </ExpandableSection>
-            <ExpandableSection
-              className="review-expandable-section"
-              toggleText={<Title headingLevel="h3">{_("Installation destination")}</Title>}
-              onToggle={() => setShowInstallationDestSection(!showInstallationDestSection)}
-              isExpanded={showInstallationDestSection}
-              isIndented
-            >
+            <ReviewDescriptionList>
+                <DescriptionListGroup>
+                    <DescriptionListTerm>
+                        {_("Operating system")}
+                    </DescriptionListTerm>
+                    <DescriptionListDescription id={idPrefix + "-target-operating-system"}>
```

Comment:
I know it's in the designs but this is inm y opinion not needed here. We mention in a huge title on the top of the wizard in all pages which OS is going to be installed.

Reply:
I agree but I thought why not :) There is more than enough space on the page. Let's see if @garrett has a strong option about this.